lib/screens/surveydetails/widget/question_paging_widget.dart

```diff
@@ -158,6 +161,29 @@ class _QuestionPagingWidgetState extends State<QuestionPagingWidget> {
     );
   }
 
+  Widget _buildAnswerContent(BuildContext context, QuestionUiModel question) {
+    return Expanded(
+      child: _buildAnswerWidget(question),
+    );
+  }
+
+  Widget _buildAnswerWidget(QuestionUiModel question) {
+    switch (question.displayType) {
+      case DisplayType.star:
+        return _answerEmojiRatingWidget(question, '⭐️');
+      default:
+        return const SizedBox();
+    }
+  }
+
+  Widget _answerEmojiRatingWidget(QuestionUiModel question, String emoji) {
```

Comment:
I think the convention is to use a verb at the start of the name 😇 
```suggestion
  Widget _buildAnswerEmojiRatingWidget(QuestionUiModel question, String emoji) {
```